DOCS-128 - Clean up screenshots for Cloud SIEM (#3990)
SumoLogic/sumologic-documentation: docs/cse/ingestion/ingestion-sources-for-cloud-siem/corelight-zeek.md

In this step, you configure a Sumo Logic Ingest Mapping in Cloud SIEM for the source category assigned to your source or collector you configured in [Step 1](#step-1-configure-collection). The mapping tells Cloud SIEM the information it needs to select the right mapper to process messages that have been tagged with that source category. 

> 1. Click **Configuration** in the top menu, and under **Integrations** select **Sumo Logic** .

Reviewer: ```suggestion
1. From in the top menu, click **Configuration**. Under **Integrations**, select **Sumo Logic**.
```

1. On the **Sumo Logic Ingest Mappings** page, click **Create**.<br/><img src={useBaseUrl('img/cse/ingest-mappipngs.png')} alt="Sumo Logic ingest mappings" style={{border: '1px solid gray'}} width="800"/>
1. On the **Create Sumo Logic Mapping** popup:
    1. **Source Category**. Enter the category you assigned to the HTTP Source or Hosted Collector in [Step 1](#step-1-configure-collection). 
    1. **Format**. Enter *Bro/Zeek JSON*.  
    1. **Event ID**. *`{_path}`*.<br/><img src={useBaseUrl('img/cse/corelight-edit-mapping.png')} alt="Corelight edit mappings" style={{border: '1px solid gray'}} width="400"/> 
1. Click **Create** to save the mapping.

## Step 4: Verify Ingestion

In this step, you verify that your logs are successfully making it into Cloud SIEM. 

> 1. Click **Configuration** in the top menu, and under **Incoming Data** select **Log Mappings**.

Reviewer: Hi @jpipkin1 ! For these changes,

```suggestion
1. From the top menu, click **Configuration**. Under **Incoming Data**, select **Log Mappings**.
```
